root_orchestrator/resource-abstractor/db/clusters_db.py

```diff
@@ -13,14 +30,58 @@ def find_clusters(filter):
             }
         },
     ]
-    return db.mongo_clusters.db.clusters.aggregate(pipeline)
+    return db.mongo_clusters.aggregate(pipeline)
 
 
 def find_cluster_by_id(cluster_id):
     cluster = list(find_clusters({"_id": ObjectId(cluster_id)}))
     return cluster[0] if cluster else None
 
 
-def find_cluster_by_name(name):
-    cluster = list(find_clusters({"cluster_name": name}))
-    return cluster[0] if cluster else None
+def update_cluster_information(cluster_id, data):
+    """Save aggregated Cluster Information"""
+
+    datetime_now = datetime.now()
+    datetime_now_timestamp = datetime.timestamp(datetime_now)
+
+    cpu_percent = data.get("cpu_percent")
+    cpu_cores = data.get("cpu_cores")
+    memory_percent = data.get("memory_percent")
+    memory_in_mb = data.get("cumulative_memory_in_mb")
+    nodes = data.get("number_of_nodes")
+    gpu_cores = data.get("gpu_cores")
+    gpu_percent = data.get("gpu_percent")
+    # technology = data.get('technology')
```

Comment:
Nitpick: Could you add a comment to this commented in line explaining why it is commented it?
Can be helpful when you or someone else returns to this in the future, otherwise, this will become guesswork.

Reply:
Ok than its fine for now, I guess.